components/ILIAS/Notifications/classes/Setup/PushNotificationObjective.php

```diff
@@ -29,24 +29,13 @@
 
 class PushNotificationObjective extends BuildArtifactObjective
 {
-    public function __construct(protected readonly Config $config)
+    public function __construct(protected readonly array $provider, protected readonly Config $config)
```

Comment:
Please define the shape of `$provider`, this should be a list of providers: `list<\ILIAS\Notifications\Interfaces\PushProviderInterface>` I guess.